packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -338,6 +338,17 @@ export interface PluginConfigTypeIos {
    * and [Apple's documentation on Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
    */
   privacyManifestAggregationEnabled?: boolean;
+
+  /**
+   * Enable support for prebuilt React Native iOS dependencies (ReactNativeDependencies.xcframework).
+   * This feature is only available from React Native 0.80.
+   * `false` will set `ENV['RCT_USE_RN_DEP'] = '1'` in the Podfile to use prebuilt third-party dependencies.
```

Comment:
```suggestion
   * When set to `false`, it will set `ENV['RCT_USE_RN_DEP'] = '1'` in the Podfile to use prebuilt third-party dependencies.
```

Let's improve this for readability.